operator/watchers/pod.go

```diff
@@ -47,15 +48,26 @@ var (
 	UnmanagedPodStoreSynced = make(chan struct{})
 )
 
+// podIpIndexFunc indexes pods by IP address.
+func podIpIndexFunc(obj interface{}) ([]string, error) {
```

Comment:
nit: do we want/need `Func` in the name of a `func`?

Reply:
This seems to be a pattern followed in `client-go` https://github.com/kubernetes/client-go/blob/master/tools/cache/index.go#L86-L92
Likely because the function is designed to be passed around.

Reply:
OK, fair enough.